assets/espresso/cp.j2.in

```diff
@@ -0,0 +1,43 @@
+{% if subworkflowContext.MATERIAL_INDEX %}
+{%- set input = input.perMaterial[subworkflowContext.MATERIAL_INDEX] -%}
+{% endif -%}
+&CONTROL
+    calculation = 'cp'
+    restart_mode = '{{ input.RESTART_MODE }}'
+    tstress = .true.
+    tprnfor = .true.
+    outdir = {% raw %}'{{ JOB_WORK_DIR }}'{% endraw %}
+    prefix = 'cp'
```

Comment:
**NOTE:** setting `prefix` same as template name, so that we will have all `cp` input/output files have the same basename, which helps load data using `dpdata`. Check if there could be any side-effects that we need to consider 🤔